Marlin/Configuration.h

```diff
@@ -646,6 +646,8 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
   #define EEPROM_CHITCHAT // Please keep turned on if you can.
 #endif
 
+//#define HOST_KEEPALIVE_FEATURE // Send messages to the host when busy with long operations
```

Comment:
If Wackerbarth would be around, he'd complain about: "Don't enable anything by default."
How about: //#define DISABLE_HOST_KEEPALIVE_FEATURE ?